composer.json

```diff
@@ -129,7 +130,12 @@
             "docroot/libraries/{$name}": ["type:drupal-library", "type:bower-asset", "type:npm-asset"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
         },
-        "patches": {}
+        "enable-patching": true,
+        "patches": {
+            "drupal/config_ignore": {
+                "Offset error within IgnoreFilter::activeReadMultiple() - https://www.drupal.org/project/config_ignore/issues/2972302": "https://www.drupal.org/files/issues/2018-06-05/offset-error-within-2972302-6.patch"
```

Comment:
@sonnykt  As per https://www.drupal.org/project/config_ignore/issues/2972302, patch number #13 and not #6 has been merged to dev. Should we not use 
https://www.drupal.org/files/issues/2018-07-31/offset-error-within-2972302-13.patch in that case?

Reply:
@gargsuchi you are right, I used an old patch. I updated the PR with the new one.